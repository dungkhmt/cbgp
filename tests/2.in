50 50
50 125
10 5
14 5
31 7
5 12
18 13
11 44
1 18
31 43
43 5
46 14
21 13
1 8
16 12
20 50
43 1
14 36
27 4
2 1
24 48
12 12
11 4
15 8
13 29
4 1
32 33
43 20
40 41
12 26
34 10
45 30
19 19
11 50
4 35
8 18
3 20
27 6
6 32
16 17
4 6
5 7
7 14
48 44
40 14
13 13
1 12
5 3
2 14
2 8
1 30
44 43
31 39
30 18
8 23
9 15
40 46
43 49
1 13
15 4
12 17
14 33
5 20
9 19
30 30
15 23
31 47
7 7
49 4
25 50
7 39
24 6
1 19
5 1
9 42
17 26
47 43
2 30
6 8
17 17
2 12
12 36
2 36
6 2
41 7
19 18
44 33
9 8
10 3
32 2
18 42
43 7
1 3
43 50
10 20
5 34
9 4
7 28
40 7
37 17
15 27
43 4
20 25
24 38
46 7
19 22
1 20
9 18
