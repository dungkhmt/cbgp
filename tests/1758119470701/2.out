(29, 15), (19, 0), (5, 7), (23, 12), (31, 43), (25, 26), (43, 27), (19, 2), (8, 10), (8, 12), (24, 1), (9, 22), (36, 12), (0, 28), (10, 11), (17, 34), (9, 10), (30, 25), (39, 35), (10, 20), (40, 0), (40, 44), (23, 26), (9, 46), (2, 4), (29, 38), (48, 8), (0, 1), (2, 45), (29, 3), (32, 48), (2, 8), (2, 26), (4, 35), (17, 49), (9, 43), (19, 8), (5, 3), (4, 39), (10, 18), (0, 5), (4, 14), (8, 32), (21, 1), (13, 4), (49, 10), (29, 40), (25, 45), (12, 26), (12, 30), (3, 0), (45, 26), (20, 28), (17, 20), (39, 42), (46, 1), (1, 10), (17, 0), (22, 2), (20, 49), (24, 9), (22, 39), (48, 18), (47, 10), (36, 25), (36, 26), (35, 9), (23, 8), (9, 2), (1, 17), (11, 3), (10, 48), (40, 7), (29, 33), (17, 10), (2, 6), (3, 44), (32, 18), (10, 29), (38, 44), (15, 0), (10, 3), (5, 44), (38, 3), (0, 10), (41, 37), (13, 9), (9, 6), (23, 2), (34, 28), (14, 2), (34, 20), (27, 9), (8, 0), (27, 6), (15, 37), (33, 38), (0, 20), (2, 39), (8, 18), (15, 16), (27, 31), (24, 46), (25, 12), (12, 27), (9, 12), (0, 2), (28, 17), (40, 38), (1, 9), (30, 36), (3, 8), (22, 35), (42, 4), (10, 12), (42, 14), (46, 13), (18, 12), (16, 37), (47, 0), (40, 5), (15, 47), (10, 15), 
numberIntersection=0 
minAngle=0.04046423429435464 
sumAngle=48.42079177027348 
minEdgeLength=1.0 
sumEdgeLength=5644.887112312521 
minDistance=0.38052119532359524 
sumDistance=365435.4698508634 

0.0000000000 0.0404642343 48.4207917703 1.0000000000 5644.8871123125 0.3805211953 365435.4698508634
0: (111, 192),
1: (128, 90),
2: (78, 66),
3: (43, 21),
4: (99, 109),
5: (41, 35),
6: (113, 76),
7: (30, 38),
8: (53, 27),
9: (116, 85),
10: (178, 17),
11: (53, 15),
12: (141, 38),
13: (100, 109),
14: (96, 95),
15: (200, 9),
16: (124, 1),
17: (132, 95),
18: (137, 33),
19: (73, 68),
20: (142, 100),
21: (120, 89),
22: (104, 89),
23: (68, 41),
24: (121, 91),
25: (82, 62),
26: (74, 45),
27: (120, 69),
28: (135, 106),
29: (22, 8),
30: (84, 54),
31: (115, 78),
32: (135, 29),
33: (27, 20),
34: (137, 101),
35: (105, 92),
36: (81, 51),
37: (38, 0),
38: (29, 23),
39: (100, 92),
40: (14, 50),
41: (0, 18),
42: (97, 95),
43: (116, 79),
44: (35, 29),
45: (80, 62),
46: (108, 124),
47: (185, 22),
48: (140, 25),
49: (139, 95),
111 192 128 90 78 66 43 21 99 109 41 35 113 76 30 38 53 27 116 85 178 17 53 15 141 38 100 109 96 95 200 9 124 1 132 95 137 33 73 68 142 100 120 89 104 89 68 41 121 91 82 62 74 45 120 69 135 106 22 8 84 54 115 78 135 29 27 20 137 101 105 92 81 51 38 0 29 23 100 92 14 50 0 18 97 95 116 79 35 29 80 62 108 124 185 22 140 25 139 95 
0: (111, 192), 1: (128, 90), 2: (78, 66), 3: (43, 21), 4: (99, 109), 5: (41, 35), 6: (113, 76), 7: (30, 38), 8: (53, 27), 9: (116, 85), 10: (178, 17), 11: (53, 15), 12: (141, 38), 13: (100, 109), 14: (96, 95), 15: (200, 9), 16: (124, 1), 17: (132, 95), 18: (137, 33), 19: (73, 68), 20: (142, 100), 21: (120, 89), 22: (104, 89), 23: (68, 41), 24: (121, 91), 25: (82, 62), 26: (74, 45), 27: (120, 69), 28: (135, 106), 29: (22, 8), 30: (84, 54), 31: (115, 78), 32: (135, 29), 33: (27, 20), 34: (137, 101), 35: (105, 92), 36: (81, 51), 37: (38, 0), 38: (29, 23), 39: (100, 92), 40: (14, 50), 41: (0, 18), 42: (97, 95), 43: (116, 79), 44: (35, 29), 45: (80, 62), 46: (108, 124), 47: (185, 22), 48: (140, 25), 49: (139, 95), 
numberIntersection=0 minAngle=0.04046423429435464 sumAngle=48.4207917701236 minEdgeLength=1.0 sumEdgeLength=5644.88711231253 minDistance=0.38052119532359524 sumDistance=365435.4698508719 
Time taken: 65.557 s
